### DISCUSSION TITLE
Disable nextjs spinner in dev

### DISCUSSION THREAD

**Discussion by Sangdt** [vercel/next.js#18616] [Help]
Is there any way we can do that? Like an option in nextjs config to disable the spinner since i’m going to make a spinner at the same position too so it kinda hard to work with.

**Comment by weyert** [comment]
You probably mean the icon in the bottom right corner of the page? If so, you can have a look at these options:
https://nextjs.org/docs/api-reference/next.config.js/static-optimization-indicator
